RecoLocalMuon/RPCRecHit/src/RPCRecHitProducer.cc

```diff
@@ -74,17 +71,15 @@ RPCRecHitProducer::RPCRecHitProducer(const ParameterSet& config)
         DeadVec.push_back(Item);
     }
     inputFile.close();
+  } else {
+    theReadoutDeadStripsToken = esConsumes();
   }
 }
 
 void RPCRecHitProducer::beginRun(const edm::Run& r, const edm::EventSetup& setup) {
   // Getting the masked-strip information
   if (maskSource_ == MaskSource::EventSetup) {
-    edm::ESHandle<RPCMaskedStrips> readoutMaskedStrips;
-    setup.get<RPCMaskedStripsRcd>().get(readoutMaskedStrips);
-    const RPCMaskedStrips* tmp_obj = readoutMaskedStrips.product();
-    theRPCMaskedStripsObj->MaskVec = tmp_obj->MaskVec;
-    delete tmp_obj;
```

Comment:
I wonder how this piece has ever worked, because the `delete` should lead two double-delete as framework owns the object pointed-to by `ESHandle` (and `ESHandle::product()`).